src/features/goblins/bank/lib/bankUtils.ts

```diff
@@ -74,6 +75,10 @@ export function canWithdraw({ item, game }: CanWithdrawArgs) {
     return false;
   }
 
+  if (item in TOOLS) {
```

Comment:
this file is from #1579

Reply:
Thanks for spotting that!  I remove this change from this PR. I shouldn't be working on 2 branches at the same time 😅